cardano-cli/test/cardano-cli-test-lib/Test/Cardano/CLI/Hash.hs

```diff
@@ -51,6 +53,18 @@ exampleAnchorDataIpfsHash, exampleAnchorDataIpfsHash2 :: String
 exampleAnchorDataIpfsHash = "QmbL5EBFJLf8DdPkWAskG3Euin9tHY8naqQ2JDoHnWHHXJ"
 exampleAnchorDataIpfsHash2 = "QmdTJ4PabgSabg8K1Z4MNXnSVM8bjJnAikC3rVWfPVExQj"
 
+-- | Tamper with the base16 hash by adding one to the first character
+tamperBase16Hash :: String -> Maybe String
```

Comment:
This could be a hedgehog generator.

Reply:
I will leave it for now, because that would be more for testing the actual hash comparison, which was implemented by someone else. Maybe in another PR

Reply:
Maybe it is a good tool to add to hedgehog extras though